From 6f212911133f34a058fb217e0b090a4e2c1b4db6 Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Wed, 13 Aug 2014 19:23:59 -0400
Subject: [x86] perf: Add model number for Avoton Silvermont

Message-id: <20140813192359.25750.19904.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 87685
O-Subject: [PATCH RHEL7.1 BZ1086821 3/3] perf/x86/intel: Add model number for Avoton Silvermont
Bugzilla: 1086821
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1086821

Description:
------------

Signed-off-by: Yan, Zheng <zheng.z.yan@intel.com>
Cc: a.p.zijlstra@chello.nl
Cc: eranian@google.com
Cc: ak@linux.intel.com
Link: http://lkml.kernel.org/r/1379837953-17755-1-git-send-email-zheng.z.yan@intel.com
Signed-off-by: Ingo Molnar <mingo@kernel.org>

RHEL Version Found:
-------------------
7.0

Upstream Status:
----------------
http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=cf3b425dd8d99e01214515a6754f9e69ecc6dce8

----------------
Steve Best

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kernel/cpu/perf_event_intel.c b/arch/x86/kernel/cpu/perf_event_intel.c
index bf2830b..1ae7246 100644
--- a/arch/x86/kernel/cpu/perf_event_intel.c
+++ b/arch/x86/kernel/cpu/perf_event_intel.c
@@ -2320,6 +2320,7 @@ __init int intel_pmu_init(void)
   break;
 
  case 55: /* Atom 22nm "Silvermont" */
+ case 77: /* Avoton "Silvermont" */
   memcpy(hw_cache_event_ids, slm_hw_cache_event_ids,
    sizeof(hw_cache_event_ids));
   memcpy(hw_cache_extra_regs, slm_hw_cache_extra_regs,
-- 
1.7.1